app/src/main/java/org/oppia/app/profile/AddProfileActivityPresenter.kt

```diff
@@ -83,6 +82,11 @@ class AddProfileActivityPresenter @Inject constructor(
     binding.addProfileActivityInfoImageView.setOnClickListener {
       showInfoDialog()
     }
+    val toolbar = activity.findViewById<View>(R.id.add_profile_activity_toolbar) as Toolbar
```

Comment:
You can directly use binding.addProfileActivityToolbar instead of defining toolbar.